code/game/objects/items/toys.dm

```diff
@@ -1075,6 +1075,34 @@
 	item_state = "[colorvar]squid"
 	desc = "A small, cute and loveable squid friend. This one is in [colorvar]."
 
+//Fox Plushies
+
+
+/obj/item/toy/plushie/fox
```

Comment:
```suggestion
// Fox Plushies
/obj/item/toy/plushie/fox
```

Reply:
Couldn't we just have all the colors selectable in the loadout
Rather then having red or leaving it to chance if you get the color you want